cmd/countPairOfBases/countPairOfBases.go

```diff
@@ -4,35 +4,25 @@ import (
 	"flag"
 	"fmt"
 	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/convert"
 	"github.com/vertgenlab/gonomics/dna"
 	"github.com/vertgenlab/gonomics/exception"
 	"github.com/vertgenlab/gonomics/fasta"
 	"github.com/vertgenlab/gonomics/fileio"
 	"log"
-	"os"
-	"path/filepath"
```

Comment:
So many times we've imported the path package only to remove it in a later PR haha. You're at least the third person after Christi and me to do this haha